src/main/java/Event.java

```diff
@@ -0,0 +1,6 @@
+public class Event extends Task {
+    private static final String TYPE = "E";
+    Event(String input) {
```

Comment:
Similar inheritance files have different file structures (i.e. new line placement). Try to keep it consistent across files!